app/assets/javascripts/application/ui.js

```diff
@@ -0,0 +1,32 @@
+(function() {
```

Comment:
do we need both `application/ui.js` and `application/accounts-ui.js`?

Reply:
They do seem very similar, I imagine they could be combined... I might leave that one for you though :)